controllers/mirroring/mirroring_controller.go

```diff
@@ -355,7 +356,7 @@ func (r *MirroringReconciler) reconcileBlockPoolMirroring(
 				cephBlockPool := blockPoolByName[blockPoolName]
 
 				mirroringToken := response.BlockPoolsInfo[i].MirroringToken
-				secretName := fmt.Sprintf("rbd-mirroring-token-%s", util.CalculateMD5Hash(blockPoolName))
+				secretName := fmt.Sprintf("%s-%s", MirroringSecretPrefix, util.CalculateMD5Hash(blockPoolName))
```

Comment:
If we need to export the functionality of creating a mirroring secret name then we should export it as a function and not as a prefix. This mean the change will become:
```suggestion
				secretName := GetMirroringSecretName(blockPoolName)
```